stdlib/LinearAlgebra/test/svd.jl

```diff
@@ -29,6 +29,8 @@ using LinearAlgebra: BlasComplex, BlasFloat, BlasReal, QRPivoted
     # matrices from the factorization as expected.
     @test sf1.U*Diagonal(sf1.S)*sf1.Vt' ≊ m1
     @test sf2.U*Diagonal(sf2.S)*sf2.Vt' ≊ m2
+    
```

Comment:
The indentation here is making CI fail, as it is superfluous. Extra white spaces are not allowed as they make diffs harder to read.